inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/openapi/DataProxyController.java

```diff
@@ -52,15 +54,9 @@ public class DataProxyController {
      * GET is used for quick lookup of IP lists (e.g. via browser requests).
      */
     @RequestMapping(value = "/dataproxy/getIpList", method = {RequestMethod.GET, RequestMethod.POST})
```

Comment:
please ensure this request method. It should be one of get or post。if it's a get request, can not use @Requestbody. I think it may be get method

Reply:
Good point. Some of the legacy modules still use post let me see whether to update them all.